Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalRequest.h

```diff
@@ -166,6 +166,7 @@ typedef NS_ENUM(NSInteger, BTPayPalRequestUserAction) {
 /**
  Optional: Offers PayPal Credit if the customer qualifies. Defaults to false. Only available with PayPal Checkout and PayPal Billing Agreement.
  */
+// Can we remove this?
```

Comment:
Does the Pay Later offering intend to completely replace PayPal credit?

Reply:
@demerino The doc string for `offerCredit` says that it applies to both checkout and billing agreement flows, whereas `offerPayLater` says that it only applies to checkout. Should we keep `offerCredit` just for the billing agreement flow?

Reply:
Verified with David that we can remove `offerCredit`.

Reply:
Addressed in #618